### DISCUSSION TITLE
Firebase Auth Issue. I have been getting this issue, please i need help resolving this

### DISCUSSION THREAD

**Discussion by Emperorchris** [firebase/flutterfire#13257] [Q&A]
Running pod install...                                             72.2s
Running Xcode build...                                                  
Xcode build done.                                           223.0s
Failed to build iOS app
Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/Auth/Auth.swift:287:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:42:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/devios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:68:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:146:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:183:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:213:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:238:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/devios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/User/User.swift:98:2

Swift Compiler Error (Xcode): Expected declaration
/Users/user/dev/ios/Pods/FirebaseAuth/FirebaseAuth/Sources/Swift/User/User.swift:140:2

Encountered error while building for device.

**Comment by les-holikov** [comment]
Have same error.
Someone pls?

**Reply by Henryderson** [reply]
Were you able to resolve the issue? If yet how?

**Comment by Emperorchris** [comment]
downgrade your firebase package

**Reply by olostan** [reply]
how? To which version? I have exactly same issue

**Reply by Henryderson** [reply]
Hello pls how were you able to resolve this issue, can you help pls.

**Reply by Henryderson** [reply]
Sorry what I actually meant was what version worked for you?

**Reply by les-holikov** [reply]
My Monterey couldn't update itself due to the device being outdated, so I installed Sonoma from a bootable drive, downloaded a more recent version of Xcode, and encountered the same issue when trying to use FirebaseAuth. However, this time, changing the target iOS version to 13 worked, whereas nothing at all helped when I was on Monterey

**Comment by ctrlVnt** [comment]
As I answered on #13505 it's the Xcode version, after Firebase version 10.25.0 you need at least Xcode 15.2
